atip/simulator.py

```diff
@@ -55,7 +55,12 @@ def calculate_optics(
 
     radint = at_lattice.get_radiation_integrals(twiss=twiss)
     logging.debug("All calculation complete.")
-    return LatticeData(twiss, beamdata.tune, beamdata.chromaticity, emitdata, radint)
+    if calc_emittance:
+        return LatticeData(
+            twiss, beamdata.tune, beamdata.chromaticity, emitdata, radint
+        )
+    else:
```

Comment:
I believe this would be clearer if you set emitdata to () by default, i.e. add an else clause to the preceding if statement.

Fewer lines of code and one less if statement.